config/settings.yml

```diff
@@ -3,16 +3,6 @@ authentication:
   # Set this in the env! The below ensures that we are un-authenticatable if we
   # forget to do this in production.
   secret: <%= SecureRandom.base64 %>
-manage_api:
-  # URL of the C# API app (manage-courses-api)
-  base_url: http://localhost:6001
-  # Auth token used between backend and manage-courses-api to authorize requests.
-  secret: banana
```

Comment:
https://github.com/DFE-Digital/teacher-training-api/blob/fe1fe8a31a6a6d4b8143520e7bc0a9328f581a3b/azure/template.json#L126-L130

can be removed as well...

Reply:
Yep